src/cdp/cdpClient.spec.ts

```diff
@@ -69,7 +71,7 @@ describe('CdpClient tests.', () => {
     await mockCdpServer.emulateIncomingMessage({id: 0, result: {}});
 
     // Assert 'cdpClient' resolved message promise.
-    chai.assert.eventually.deepEqual(commandPromise, {});
+    expect(commandPromise).to.eventually.equal({});
```

Comment:
Doesn't `equal` compare references?

Reply:
The docs use `equal` all over the place: https://www.chaijs.com/plugins/chai-as-promised/

It seems to be: `be` for `chai`, and `equal` for `chai-as-promised`.